Libraries/src/Amazon.Lambda.AspNetCoreServer.Hosting/Internal/LambdaRuntimeSupportServer.cs

```diff
@@ -36,11 +38,19 @@ public LambdaRuntimeSupportServer(IServiceProvider serviceProvider)
         /// <param name="application"></param>
         /// <param name="cancellationToken"></param>
         /// <returns></returns>
+        [RequiresUnreferencedCode("Serializes objects to Json")]
```

Comment:
check if this is still necessary

Reply:
without it, IL2026 - can break functionality when trimming application code.

Will leave, but updated comment.